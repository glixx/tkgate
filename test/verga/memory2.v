module top;

  parameter N = 32;
  reg [N-1:0] m[0:1024];
  integer i;

  initial
    begin
	$readmemh("test.mem");
	for (i = 0;i < 20;i = i + 1)
	  $display("m[%d] = %h",i,m[i]);
    end

    always @(m)
      begin
	$display("m event");
      end

endmodule
