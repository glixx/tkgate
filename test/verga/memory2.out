m[0] = 3
m[1] = 4
m[2] = 9
m[3] = 16
m[4] = 25
m[5] = 36
m[6] = 49
m[7] = 64
m[8] = 81
m[9] = x
m[10] = x
m[11] = x
m[12] = 100
m[13] = x
m[14] = x
m[15] = 99
m[16] = z
m[17] = 12
m[18] = x8
m[19] = z12
